app/controllers/films_controller.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+# This is Films Controller
```

Comment:
you dont need such comments, they are not necessary, it does not really explain anything :D

Reply:
@Goran1708 Thanks for the comment, actually I did it for rubocop because it is showing that I have to explain something about the controller